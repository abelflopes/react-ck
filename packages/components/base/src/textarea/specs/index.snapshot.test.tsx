import React from "react";
import { Textarea } from "../index";
import renderer from "react-test-renderer";

describe("snapshot Textarea", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Textarea />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
