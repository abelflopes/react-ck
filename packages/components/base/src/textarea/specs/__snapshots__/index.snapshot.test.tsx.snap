// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Textarea renders correctly 1`] = `
<textarea
  className="root standalone skin_default"
/>
`;
